src/lib/modules/Series/SeriesTitle.js

```diff
@@ -10,6 +10,11 @@ export const SeriesTitle = ({ title, modeOfIssuance }) => (
 );
 
 SeriesTitle.propTypes = {
-  title: PropTypes.string.isRequired,
-  modeOfIssuance: PropTypes.string.isRequired,
+  title: PropTypes.string,
```

Comment:
actually these are required in the `schema`, they must be there... I would revert this change maybe

Reply:
I was carried away from series minimal which actually derives from another error, #70